biogeophys/FatesHydroWTFMod.F90

```diff
@@ -1056,7 +1687,7 @@ function ftc_from_psi_tfs(this,psi) result(ftc)
 
     psi_eff = min(0._r8,psi)
 
-    ftc = max(min_ftc,1._r8/(1._r8 + (psi_eff/this%p50)**this%avuln))
+    ftc = max(min_ftc*10,1._r8/(1._r8 + (psi_eff/this%p50)**this%avuln))
```

Comment:
please add a comment describing change with name and date

Reply:
This was for test, it doesn't need to be updated.

Reply:
ok